bedrock/mozorg/templates/mozorg/contribute/tasks/taskview-base.html

```diff
@@ -51,7 +51,7 @@ <h2 class="page-title">{% block taskview_title %}{% endblock %}</h2>
   </section>
 
   <ul class="notices">
-    <li><a href="{{ url('mozorg.contact.communities.communities-landing') }}" id="communities" target="_blank">{{ _('Find your local Mozilla community.') }}</a></li>
+    <li><a href="{{ url('mozorg.contact.communities.communities-landing') }}" id="communities" target="_blank" rel="noopener noreferrer">{{ _('Find your local Mozilla community.') }}</a></li>
```

Comment:
We can safely remove `target="_blank"` here. This one is not wrapped within a translated string, so no need for a conditional.